<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="narek-MS-7917" ignored="0" name="Command line test" tests="10" failures="5" timestamp="07 Apr 2018 21:59:37 GMT" time="0.421" errors="0">
  <testcase name="testAddContract" time="0.394" classname="org.badcoding.dao.interfaces.ContractDAOTest">
    <failure type="java.lang.ExceptionInInitializerError">
      <![CDATA[java.lang.ExceptionInInitializerError
at org.badcoding.dao.implementation.ContractDAOImpl.addContract(ContractDAOImpl.java:21)
at org.badcoding.dao.interfaces.ContractDAOTest.testAddContract(ContractDAOTest.java:19)
Caused by: org.hibernate.exception.JDBCConnectionException: Error calling Driver#connect
at org.hibernate.engine.jdbc.connections.internal.BasicConnectionCreator$1$1.convert(BasicConnectionCreator.java:122)
at org.hibernate.engine.jdbc.connections.internal.BasicConnectionCreator.convertSqlException(BasicConnectionCreator.java:140)
at org.hibernate.engine.jdbc.connections.internal.DriverConnectionCreator.makeConnection(DriverConnectionCreator.java:58)
at org.hibernate.engine.jdbc.connections.internal.BasicConnectionCreator.createConnection(BasicConnectionCreator.java:75)
at org.hibernate.engine.jdbc.connections.internal.DriverManagerConnectionProviderImpl.configure(DriverManagerConnectionProviderImpl.java:106)
at org.hibernate.boot.registry.internal.StandardServiceRegistryImpl.configureService(StandardServiceRegistryImpl.java:89)
at org.hibernate.service.internal.AbstractServiceRegistryImpl.initializeService(AbstractServiceRegistryImpl.java:206)
at org.hibernate.service.internal.AbstractServiceRegistryImpl.getService(AbstractServiceRegistryImpl.java:178)
at org.hibernate.engine.jdbc.internal.JdbcServicesImpl.buildJdbcConnectionAccess(JdbcServicesImpl.java:260)
at org.hibernate.engine.jdbc.internal.JdbcServicesImpl.configure(JdbcServicesImpl.java:94)
at org.hibernate.boot.registry.internal.StandardServiceRegistryImpl.configureService(StandardServiceRegistryImpl.java:89)
at org.hibernate.service.internal.AbstractServiceRegistryImpl.initializeService(AbstractServiceRegistryImpl.java:206)
at org.hibernate.service.internal.AbstractServiceRegistryImpl.getService(AbstractServiceRegistryImpl.java:178)
at org.hibernate.cfg.Configuration.buildTypeRegistrations(Configuration.java:1885)
at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:1843)
at org.badcoding.hibernate.logic.Database.<clinit>(Database.java:14)
... 31 more
Caused by: java.sql.SQLException: Access denied for user ''@'localhost' (using password: NO)
at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1084)
at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:4232)
at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:4164)
at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:926)
at com.mysql.jdbc.MysqlIO.proceedHandshakeWithPluggableAuthentication(MysqlIO.java:1748)
at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1288)
at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2506)
at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2539)
at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2321)
at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:832)
at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
at com.mysql.jdbc.Util.handleNewInstance(Util.java:409)
at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:417)
at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:344)
at org.hibernate.engine.jdbc.connections.internal.DriverConnectionCreator.makeConnection(DriverConnectionCreator.java:55)
... 44 more
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- testAddContract -->
  <testcase name="testGetContractById" time="0.001" classname="org.badcoding.dao.interfaces.ContractDAOTest">
    <failure type="java.lang.NoClassDefFoundError" message="Could not initialize class org.badcoding.hibernate.logic.Database">
      <![CDATA[java.lang.NoClassDefFoundError: Could not initialize class org.badcoding.hibernate.logic.Database
at org.badcoding.dao.implementation.ContractDAOImpl.getContractById(ContractDAOImpl.java:64)
at org.badcoding.dao.interfaces.ContractDAOTest.testGetContractById(ContractDAOTest.java:50)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- testGetContractById -->
  <testcase name="testListContracts" time="0.0" classname="org.badcoding.dao.interfaces.ContractDAOTest">
    <failure type="java.lang.NoClassDefFoundError" message="Could not initialize class org.badcoding.hibernate.logic.Database">
      <![CDATA[java.lang.NoClassDefFoundError: Could not initialize class org.badcoding.hibernate.logic.Database
at org.badcoding.dao.implementation.ContractDAOImpl.listContracts(ContractDAOImpl.java:81)
at org.badcoding.dao.interfaces.ContractDAOTest.testListContracts(ContractDAOTest.java:56)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- testListContracts -->
  <testcase name="testRemoveContract" time="0.001" classname="org.badcoding.dao.interfaces.ContractDAOTest">
    <failure type="java.lang.NoClassDefFoundError" message="Could not initialize class org.badcoding.hibernate.logic.Database">
      <![CDATA[java.lang.NoClassDefFoundError: Could not initialize class org.badcoding.hibernate.logic.Database
at org.badcoding.dao.implementation.ContractDAOImpl.getContractById(ContractDAOImpl.java:64)
at org.badcoding.dao.interfaces.ContractDAOTest.testRemoveContract(ContractDAOTest.java:42)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- testRemoveContract -->
  <testcase name="testUpdateContract" time="0.0" classname="org.badcoding.dao.interfaces.ContractDAOTest">
    <failure type="java.lang.NoClassDefFoundError" message="Could not initialize class org.badcoding.hibernate.logic.Database">
      <![CDATA[java.lang.NoClassDefFoundError: Could not initialize class org.badcoding.hibernate.logic.Database
at org.badcoding.dao.implementation.ContractDAOImpl.getContractById(ContractDAOImpl.java:64)
at org.badcoding.dao.interfaces.ContractDAOTest.testUpdateContract(ContractDAOTest.java:35)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- testUpdateContract -->
  <testcase name="testAddCustomer" time="0.0" classname="org.badcoding.dao.interfaces.CustomerDAOTest"/>
  <testcase name="testGetCustomerById" time="0.001" classname="org.badcoding.dao.interfaces.CustomerDAOTest"/>
  <testcase name="testListCustomers" time="0.0" classname="org.badcoding.dao.interfaces.CustomerDAOTest"/>
  <testcase name="testRemoveCustomer" time="0.0" classname="org.badcoding.dao.interfaces.CustomerDAOTest"/>
  <testcase name="testUpdateCustomer" time="0.001" classname="org.badcoding.dao.interfaces.CustomerDAOTest"/>
</testsuite> <!-- Command line test -->
